### PR TITLE
used twitter-stream-api node to get real time tweets

### PR DESCRIPTION
**Pull request by samitsv**
- used new twitter stream node used for getting real time tweets

### CODE REVIEW

File: controller/twitter.js [hunk 1]
```diff
@@ -1,21 +1,62 @@
 'use strict';
 
-let Twitter = require('twitter'),
+let Writable = require('stream').Writable,
```
Comment (vivek-sethia) [inline controller/twitter.js:3]: const could be used instead of let

File: controller/twitter.js [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
+    TwitterStream = require('twitter-stream-api'),
     config = require('../config/config');
```
Comment (swathi-ssunder) [inline controller/twitter.js:6]: Does this now work for both live tweets and existing tweets?

Comment (samitsv) [inline controller/twitter.js:6]: the new node acts in the same way as the old one did, except it takes into account the real time tweets as well

File: controller/twitter.js (hunk 1 above)
Comment (samitsv) [inline controller/twitter.js:3]: made the changes

**Pull request merged**